packages/components/src/templates/next/layouts/Collection/CollectionClient.tsx

```diff
@@ -309,7 +309,7 @@ const CollectionClient = ({
                   </>
                 )}
               </p>
-
```

Comment:
why ah, do we not want this?

Reply:
got design feedback that the pills might not be relevant because you can directly uncheck filters. it makes alignment tricky as well

but just hiding it for now because we are redesigning the layout, might end up keeping it